### ISSUE TITLE
Not able to use curl inside image

### ISSUE THREAD

**Issue by anuragagarwal561994** [mesosphere/spark-build#267]
Setting `LD_LIBRARY_PATH`, is not able to use libraries from `/usr/lib` because of which `curl` is not working.

```Error: curl: /opt/mesosphere/libmesos-bundle/lib/libcurl.so.4: version `CURL_OPENSSL_3' not found (required by curl)```

https://github.com/mesosphere/spark-build/blob/c7c418d4f2138e7a0862e5d25b8b7e8fa960e393/docker/Dockerfile#L59

**Comment by hibbert** [comment]
The same thing happens in the jenkins image that mesos builds, so presumably something is wrong with the libmesos bundle also things like apt-get don't work

**Comment by antinodes** [comment]
FYI, you can workaround this by running `unset LD_LIBRARY_PATH`.  If you are deriving from this image then use `RUN unset LD_LIBRARY_PATH && curl -L -O <url>`.

**Comment by vishnu2kmohan** [comment]
@anuragagarwal561994 Please let us know if you continue to experience this issue with the [latest release of Spark 2.5.0-2.2.1](https://docs.mesosphere.com/services/spark/2.5.0-2.2.1/release-notes/)

**Comment by anuragagarwal561994** [comment]
@vishnu2kmohan Yes I used `mesosphere/spark:2.5.0-2.2.1-hadoop-2.7` docker image and it works fine.
